benchmarks/movielens.ipynb

```diff
@@ -78,15 +78,16 @@
      "text": [
```

Comment:
<p>Can you rerun this notebook to capture results from all algorithms?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/944/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
updated!